processes/cve_management_process.md

```diff
@@ -0,0 +1,123 @@
+# Node.js CVE management process
+
+The Node.js project acts as a Commonn Vulnerabilities and Exposures (CVE)
+Numbering Authority (CNA). The current scope is for all actively
+developed versions of software developed under the Node.js project (ie.
+https://github.com/nodejs).  This means that the Node.js team reviews
+CVE requests and if appropriate assigns CVE numbers to vulnerabilities.
+The scope currently **does not** include thir party modules.
```

Comment:
*Only* for Core and Foundation projects or userland modules also?

Reply:
It is only for Core and Foundation projects.  That line tries to say that userland modules are excluded for now.  Once we are comfortable with that scope we will consider expanding.

Reply:
s/thir/third